tests/Rector/MethodCall/ResponseHelperCallToJsonResponseRector/config/configured_rule.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Routing\ResponseFactory;
+use Rector\Config\RectorConfig;
+use RectorLaravel\Rector\MethodCall\ResponseHelperCallToJsonResponseRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->import(__DIR__ . '/../../../../../config/config.php');
+
+    $rectorConfig->rule(ResponseHelperCallToJsonResponseRector::class);
+};
+
+function response(): ResponseFactory
```

Comment:
Remove this declaration here please. It might conflict with existing `response()` function in the apps, since it has no checks with `function_exists()`. We don't need it for tests to pass (when we're not using the ObjectType check).

Reply:
Yes I was aware that this was kind of hacky... Was just my first approach to test it. It's removed now